apps/web/src/components/display/AddRecordModal.tsx

```diff
@@ -1,48 +1,54 @@
 import { useRouter } from 'next/router';
-import { Formik, Form as FormikForm } from 'formik';
+import { Formik, Form as FormikForm, FieldProps } from 'formik';
 import createValidator from 'class-validator-formik';
+import ReactSelect from 'react-select';
 
 import { Modal } from '../Modal';
-import { Button } from '@components';
-import { addJobRecord, useGetAddRecordOptions, RecordTypeOptions } from '@services';
-import { IENApplicantJobCreateUpdateDTO } from '@ien/common';
-import { Field, Select, Option } from '../form';
+import { Button, getSelectStyleOverride } from '@components';
+import {
+  addJobRecord,
+  useGetAddRecordOptions,
+  RecordTypeOptions,
+  updateJobRecord,
+} from '@services';
+import { ApplicantJobRO, IENApplicantJobCreateUpdateDTO } from '@ien/common';
+import { Field } from '../form';
+import dayjs from 'dayjs';
```

Comment:
just a nit, I like to keep imports organized from top to bottom, top being 3rd party, bottom being created by us